dandiapi/settings.py

```diff
@@ -77,6 +77,9 @@ def mutate_configuration(configuration: Type[ComposedConfiguration]):
     # considered a temporary fix.
     DANDI_SCHEMA_VERSION = values.Value(default='0.6.2', environ=True)
 
+    # The version of the API
+    DANDI_API_VERSION = values.Value(default='v0.0.0', environ=True)
```

Comment:
What's the purpose of having a nonsensical API version?

Reply:
Aside from production, this value really isn't important, so I set a default just to reduce any unnecessary friction. If you think it should be required I can remove the default too, I don't feel too strongly.